tests/functionality_tests/mission_test.py

```diff
@@ -0,0 +1,54 @@
+from engine.robot import Robot
+from engine.base_station import BaseStation
+from engine.mission import Mission
+from engine.mission import ControlMode
+from engine.database import DataBase
+from engine.robot_state import Robot_State
+from engine.phase import Phase
+
+import math
+
+
+class MissionTest():
+
+    def __init__(self):
+        self.rb_state = Robot_State(0, 0, 0, .2, .5, .2)
+        self.rob = Robot(self.rb_state)
+        self.rob_base = BaseStation(position=(
+            42.444250, -76.483682), heading=math.pi/2, battery=None, plastic_load=None)
+        self.database = DataBase(self.rob)
+        self.m = Mission(robot=self.rob, base_station=self.rob_base,
```

Comment:
for sake of consistency, can we have 

robot variable name r2d2 or robot_one
mission variable name mission (m is kinda confusing)